libraries/apollo-mockserver/src/commonMain/kotlin/com/apollographql/apollo3/mockserver/MockServer.kt

```diff
@@ -235,11 +249,21 @@ internal class MockServerImpl(
 }
 
 @JsName("createMockServer")
-fun MockServer(): MockServer = MockServerImpl(QueueMockServerHandler(), true, TcpServer(), null)
+@JvmOverloads
+fun MockServer(port: Int = 0): MockServer = MockServerImpl(
```

Comment:
Heads up I have reverted this change because `MockServer(Int)` was a thing in 3.x and this would make tests fail without any build warning and/or error when upgrading to 4.0. 

Since we have a builder, configuring the port can be done with:

```kotlin
    val mockServer = MockServer.Builder()
        .tcpServer(TcpServer(port = port))
        .build()
```

If this is deemed too verbose, we can add `Builder.port()` as a shorthand but I'd like us to try without first since it's always easier to add than remove.